hometown/Huberts_Vienna_Tips.md

```diff
@@ -0,0 +1,16 @@
+If you visit Vienna you should see the following:
+
+## Places to eat
+- Cafe Schwarzenberg
+- Cafe Demel
+- Plachuta
```

Comment:
This is a great place to eat.  There are two Plachutas in Vienna.  Please add the addresses.